app/build.gradle

```diff
@@ -27,4 +48,7 @@ dependencies {
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
     implementation 'com.android.support:support-v4:28.0.0'
     implementation 'com.android.support:design:28.0.0'
+    compile('com.crashlytics.sdk.android:crashlytics:2.9.5@aar') {
```

Comment:
what is the difference between compile and implementation, can you prove why do we need to use here compile instead of implementation?

Reply:
Compile allows my application to access crashlytics library, as shown in MainActivity:
Fabric.with(this, new Crashlytics());

Using implementation no classes from that library are exposed

Reply:
Changed to implementation